game/views.py

```
@@ -1488,7 +1518,25 @@ def lesson_detail_view(request, lesson_name):
+                },
+                {
+                    "instruction": "Move the queen from h5 to f7 and deliver checkmate.",
+                    "expected_move": "h5-f7"
+                }
+            ],
+
+            "practice_position": {
+                "h5": "Q",
+                "e8": "K",
+                "c4": "B",
+                "e8": "K",
```

Ruff (0.15.15)

[error] Dictionary key literal `"e8"` repeated

Remove repeated key literal `"e8"`

(F601)